### DISCUSSION TITLE
How do you hide left pane?

### DISCUSSION THREAD

**Discussion by J-Kappes** [gokcehan/lf#1643] [Q&A] [answered]
How do you hide the sibling directories?

**Comment by DusanLesan** [comment, marked as answer]
You control other panels by [setting ratios](https://github.com/gokcehan/lf/blob/master/doc.md#ratios-int-default-123) for them. You can have 4 of them with `set ratios 1:1:1:1` or two by `set ratios 1:2:3` . If you want to have only one of them, you must first disable preview with `set preview false` and then calling `set ratios 1`.
